lib/routes/cctv/xwlb.ts

```diff
@@ -75,14 +75,21 @@ const getXWLB = async () => {
                     const res = await got(url);
                     const content = load(res.data);
                     const list: string[] = [];
-                    content('body li').map((i, elem) => {
-                        const e = content(elem);
-                        const href = e.find('a').attr('href');
-                        const title = e.find('a').attr('title');
-                        const dur = e.find('span').text();
-                        list.push(`<a href="${href}">${title} ⏱${dur}</a>`);
-                        return i;
-                    });
+                    await Promise.all(
```

Comment:
This dramatically increase the number of HTTP requests required [from 21 to 321](https://github.com/DIYgod/RSSHub/actions/runs/12961539745/job/36156928438#step:13:18) which is a big NO.